client/src/hooks/useSearch.js

```diff
@@ -0,0 +1,85 @@
+import { useRouter } from 'next/router'
+
+export const useSearch = (state, setState) => {
+  const search = state
+  const setSearch = setState
+  const router = useRouter()
+
+  const goToSearchResults = (goToStart = false) => {
+    if (goToStart) delete search.query.offset
+    const { pathname, query } = search
+    router.push({ pathname, query })
+  }
+
+  const setSearchString = (searchString, resetQuery = false) => {
+    // should this clear the search facets?
```

Comment:
I would imagine so right? Couldn't the user could have potentially selected some facets that aren't even in the results of the new search?

Reply:
I think it might be more complicated, so I made it an option since it might change depending on context... like if you wanted to use this in a component that searched users that you could filter by organization..
You might want to change the name but keep the organization facet in the search query

Reply:
Makes sense. A default of true might make sense though right?